v2/fleet/aws-ecr-login.service

```diff
@@ -0,0 +1,26 @@
+[Unit]
+Description=Amazon ECR Login
+After=docker.service
+Requires=docker.service
+Requires=network-online.target
+After=network-online.target
+
+[Service]
+Type=oneshot
+User=core
+Environment="AWS_ACCOUNT_ID=etcdctl get /ECR/registry-account"
```

Comment:
Can we use an ECR-specific name for this env var so it's clear what we're using it for? (i.e. ECR_AWS_ACCOUNT_ID)